src/main/java/hudson/remoting/RemoteClassLoader.java

```diff
@@ -1037,7 +1039,7 @@ public String toString() {
          * any new classpath. In this way, we can effectively use this classloader as a representation
          * of the bootstrap classloader.
          */
-        private static final ClassLoader PSEUDO_BOOTSTRAP = new URLClassLoader(new URL[0],(ClassLoader)null) {
+        private static final ClassLoader PSEUDO_BOOTSTRAP = new URLClassLoader(new URL[0], null) {
```

Comment:
Might be a topic for the future: add Platform classloader here as in other Java 11 fixes. CC @jenkinsci/java11-support .
Otherwise it *might* be loading platform classes from the master on Java 11

Reply:
That's a interesting point.

Reply:
I read recently that most classes in Java are loaded by a proper `ClassLoader` now, though it'll be a while until they can fully replace the bootstrap `null` reference with the platform `ClassLoader` everywhere in the JDK (one day!)